src/App/Pages/LoginTwoFactorPage.cs

```diff
@@ -254,6 +260,7 @@ private void Init()
 
                 table.WrappingStackLayout = () => layout;
                 scrollView.Content = layout;
+                ToolbarItems.Add(continueToolbarItem);
```

Comment:
Same

Reply:
GitHub wants me to comment here as well. Same as above, USB entry does work on Android so I think it makes sense to leave it in. If you really want I can remove it on Android though.